modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -8937,12 +8937,18 @@ export const console: ConsoleNS = {
                         }
                     },
                     roles: {
+                        placeHolders: {
+                            emptyListPlaceholder: {
+                                subtitles: "There are no roles assigned to this group at the moment.",
```

Comment:
```suggestion
                                subtitle: "There are no roles assigned to this group at the moment.",
```

Reply:
This attribute is coming from the `Placeholder` interface, which accepts both an object and a string.

- https://github.com/wso2/identity-apps/blob/ce1a76c37419a5f0019956078b538c2517db7fb8/modules/i18n/src/models/common.ts#L81